semantic-conventions/src/opentelemetry/semconv/model/semantic_attribute.py

```diff
@@ -74,6 +76,14 @@ def import_attribute(self):
     def inherit_attribute(self):
         return replace(self, inherited=True)
 
+    @property
+    def core_type(self):
```

Comment:
If we are using template terminology then maybe `get_instantiated_type`? With "core", I would expect to get `int` for `int[]`, and what about enums?

Reply:
done